src/Main.java

```diff
@@ -1,15 +1,87 @@
-//TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
-// click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
 public class Main {
     public static void main(String[] args) {
-        //TIP Press <shortcut actionId="ShowIntentionActions"/> with your caret at the highlighted text
-        // to see how IntelliJ IDEA suggests fixing it.
-        System.out.printf("Hello and welcome!");
-
-        for (int i = 1; i <= 5; i++) {
-            //TIP Press <shortcut actionId="Debug"/> to start debugging your code. We have set one <icon src="AllIcons.Debugger.Db_set_breakpoint"/> breakpoint
-            // for you, but you can always add more by pressing <shortcut actionId="ToggleLineBreakpoint"/>.
-            System.out.println("i = " + i);
-        }
+        //Task 1
+        int a = 3000;
+        byte b = 127;
+        short c = 32000;
+        long d = 1000L;
+        float e = 13.11111111f;
+        double f = 0.123456789;
+        System.out.println("Значение переменной a с типом int равно " + a);
+        System.out.println("Значение переменной b с типом byte равно " + b);
+        System.out.println("Значение переменной c с типом short равно " + c);
+        System.out.println("Значение переменной d с типом long равно " + d);
+        System.out.println("Значение переменной e с типом float равно " + e);
+        System.out.println("Значение переменной f с типом double равно " + f);
+        //Task2
+        float a1 = 27.12f;
+        long b1 = 987678965549L;
+        float c1 = 2.786f;
+        int d1 = 569;
+        int e1 = -159;
+        short f1 = 27897;
+        byte g1 = 67;
+        //Task 3
+        int lP = 23;
+        int aS = 27;
+        int eA = 30;
+        int totalPaper = 480;
+        int totalStudents = lP + aS + eA;
+        int paperPerStudent = totalPaper / totalStudents;
+        System.out.println("На каждого ученика рассчитано " + paperPerStudent + " листов бумаги");
+        //Task 4
+        int overallPerf = 16 / 2;
+        int time1 = 20;
+        int time2 = 24 * 60;
+        int time3 = 3 * 24 * 60;
+        int time4 = 31 * 24 * 60;
```

Comment:
```suggestion
        int time4 = 31 * time2;
```
используем переменные, а не считаем заново